src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx

```diff
@@ -47,28 +48,34 @@ export default function ChooseCommunityCare() {
     { value: 'name', label: 'Name' },
   ]);
 
+  const showAlert = true;
```

Comment:
So is this hard coded to always show the alert?

Reply:
For now yes until we determine and add logic for showing the alert. This ticket was just for building out the UI

Reply:
Ah I see so this component isn't used publicly yet?

Reply:
yup